oeps/processes/oep-0014-proc-archive-repos.rst

```diff
@@ -88,6 +88,14 @@ These steps should be followed for all repos within the Open edX organization (f
 
   - Move the repository to the openedx-unsupported organization
 
+  - Remove the repo from ``openedx-translations`` and Transifex
+    
+    - Remove the repo entry from the ``extract-translation-source-files`` workflow
```

Comment:
To be clear, this workflow is in the ``openedx-translations`` repo? (and the following step same question)